jekyll/_cci2/orb-intro.md

```diff
@@ -158,7 +158,7 @@ Available Commands:
   validate    Validate an orb.yml
 ```
 
-**Note** When authoring an orb, you will agree to CircleCI's Code Sharing Terms of Service when your organization opts-in to 3rd party orb use and authoring. CircleCI thereby licenses all orbs back to users under the MIT License agreement.
+**Note** When authoring an orb, you will agree to CircleCI's [Code Sharing Terms of Service](https://circleci.com/legal/code-sharing-terms/)when your organization opts-in to 3rd party orb use and authoring. CircleCI thereby licenses all orbs back to users under the MIT License agreement.
```

Comment:
```suggestion
**Note** When authoring an orb, you will agree to CircleCI's [Code Sharing Terms of Service](https://circleci.com/legal/code-sharing-terms/) when your organization opts-in to 3rd party orb use and authoring. CircleCI thereby licenses all orbs back to users under the MIT License agreement.
```

Reply:
You'll just want a space between the inline link and `when`. Otherwise LGTM! 🚢

Reply:
Thanks @tyler for the catch. Fixed in document.